CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+will house contributing information and guidance that we provide to CNCF
+projects for contributor strategy topics. The intention is to grow this as a
+resource for all contributors - aspiring, current, and maintainers.  
+
+Our [charter] describes our mission and initial scope of what we’d like to
+accomplish. Each of our members are closely involved with at least one CNCF
+project; we are here to learn as much as those we serve.
+
+## Ground Rules
+We very much welcome everyone on any of our [communication channels] including
+through GitHub Issues and Pull Requests (PRs) and ask all contributors to
+maintain our welcoming culture. We follow the [CNCF’s code of conduct].
+
+Async communication is our preference because of our global nature but
+absolutely value face to face time and meetings when it makes sense. Slack works
+well for real time questions and discussion but we will move decisions and key
+tasks to GitHub or the mailing list. Our communication channels are on our
+[README].
+
+We don’t sponsor specific company branded documentation and resources as
+official guidance to contributors and/or projects. A resources list[link tbu]
+that may link to other sources that are not official, including company branded
+information, in the strategy repo, is strictly for our groups planning purposes
+only.
+
+## How to Help
+
+**Contributors and/or Maintainers of a CNCF incubated or graduated project:**  
+- Be a liaison for your project with our group
+  - Why? The ecosystem is very large, if you stay connected, you’ll be closer to
+resources and advice.
+  - Take back this information to build (or continue to build!) intentional
+community and governance spaces and/or roles in your project to do this work at
+an operational level
+- Join us for a Contributor Strategy AMA on the 4th Thursday meeting of each month
+ at the last :30 minutes as you need. This is your time to ask us general or
+ specific questions that can give you ideas or unblock you. The other meeting
+ time we have is devoted to meta SIG and working group planning.
+- Give us feedback on where you need help, guidance, requested templates,
+recommendations, and more here
+Or you can provide feedback by filing issues against the
+cncf/sig-contributor-strategy repo, send a note to the mailing list, or find a
+chair or working group facilitator privately on slack.
+- Become a contributor in one of our [working groups]
+  - File issues and PRs
+  - Project management
+- Create workshops, talks, blogs, and papers on related contributor content (or
+  reuse the good stuff you have!)
+- Create templates and other scalable guidance
+- Tell us what WORKS! Sounds like a lot of “what doesn’t work” above but we also
+want to know what works well for projects in their contributor and governance
+areas. Looking at you graduated projects but we see the incubation project crew
+still working on great programs!
+- Be a part of our work in progress maintainers circle! [link to issue]
```

Comment:
Do you want to link to https://github.com/cncf/sig-contributor-strategy/issues/1 here?